subprojects/logger/gradle-logger/src/main/kotlin/com/avito/logger/GradleLoggerFactory.kt

```diff
@@ -146,7 +146,6 @@ public class GradleLoggerFactory(
             verboseMode = getVerbosity(project)?.let { VerboseMode(it, doPrintStackTrace(project)) }
         )
 
-        @Suppress("UnstableApiUsage")
```

Comment:
How did you find all obsolete suppressed cases?

Reply:
Just removed all of them )